package.json

```diff
@@ -78,7 +78,7 @@
     "typescript": "^4.0.3"
   },
   "engines": {
-    "node": ">= 10.12.0"
+    "node": "^17 || ^16 || ^14 || ^12"
```

Comment:
Are you sure you want to include `^17`? Is the intent to support non-LTS versions in this module? The 17 will be added to the test matrix anyways - but it will be marked as "experimental", i.e. the general recommendation is that you should keep testing in it in preparation for 18, but you don't need to declare it as "supported" (because once you drop the support for it you'd need to bump major).

Reply:
Ah I see - good point. If the 17 is automatically included in the test matrix than I think it's better to remove it from supported.